src/ui/VideoPlayer.css

```diff
@@ -0,0 +1,44 @@
+.widget-video-player-iframe{
+    position: absolute;
+    border: 0px;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    width: 100%;
+    height: 100%;
+}
+
+.widget-video-player-html5{
+    background-color: #000;
+    position: absolute;
+}
+.video-error-label{
```

Comment:
add white-line for
use space before parentheses
`.video-error-label {`

Let have a look if there is a style validation guide for CSS. Let check with UX